modules/client-configuration/pages/supported-features-sle-micro.adoc

```diff
@@ -0,0 +1,165 @@
+Do not use this feature on production systems.
+====
+
+[IMPORTANT]
+====
+The operating system you run on a client is supported by the organization that supplies the operating system.
+{micro} is supported by {suse}.
+{opensuse}{nbsp}{microos} is supported by the {suse} community.
+====
+
+
+The icons in this table indicate:
+
+* {check} the feature is available on both Salt and traditional clients
+* {cross} the feature is not available
+* {question} the feature is under consideration, and may or may not be made available at a later date
+* {trad} the feature is supported only on traditional clients
+* {salt} the feature is supported only on Salt clients.
+
+
+[cols="1,1", options="header"]
+.Supported Features on {micro} and {opensuse}{nbsp}{microos} Operating Systems
+|===
+
+| Feature
+| {micro} and {opensuse}{nbsp}{microos}
+
+| Client
+| {salt}
+
+| Operating system packages
+| {salt}
+
+| Registration
+| {salt}
+
+| Install packages
+| {salt}
+
+| Apply patches (requires CVE ID)
+| {salt}
+
+| Remote commands
+| {salt}
+
+| System package states
+| {salt}
+
+| System custom states
+| {salt}
+
+| Group custom states
+| {salt}
+
+| Organization custom states
+| {salt}
+
+| System set manager (SSM)
+| {salt}
+
+| Service pack migration
+| N/A
+
+| Basic Virtual Guest Management {star}
+| {cross}
+
+| Advanced Virtual Guest Management {star}
+| {question}
+
+| Virtual Guest Installation (Kickstart), as Host OS
+| {cross}
+
+| Virtual Guest Installation (image template), as Host OS
+| {question}
+
+| System deployment (PXE/Kickstart)
+| {question}
+
+| System redeployment (Kickstart)
+| {cross}
+
+| Contact methods
+| {salt}: ZeroMQ
+
+| Works with {productname} Proxy
+| {salt}
+
+| Action chains
+| {question}
+
+| Staging (pre-download of packages)
+| {question}
+
+| Duplicate package reporting
+| {salt}
+
+| CVE auditing (requires CVE ID)
+| {salt}
+
+| SCAP auditing
+| {question}
+
+| Package verification
+| {question}
+
+| Package locking
+| {salt}
+
+| System locking
+| {question}
+
+| Maintenance Windows
+| {question}
+
+| System snapshot
+| {cross}
+
+| Configuration file management
+| {salt}
+
+| Snapshots and profiles
+| {salt}: Profiles supported, Sync not supported
+
+| Power management
+| {question}
+
+| Monitoring
+| {salt}
+
+| Docker buildhost
+| {cross}
+
+| Build Docker image with OS
+| {cross}
+
+| Kiwi buildhost
+| {cross}
+
+| Build Kiwi image with OS
+| {salt}
+
+| Recurring Actions
+| {salt}
+
+| AppStreams
+| N/A
+
+| Yomi
+| {question}
+
+
+{star} Virtual Guest Management:
+
+In this table, virtual guest management is split into basic and advanced.
+
+Basic virtual guest management includes listing VMs, slow refresh, VM lifecycle actions (start, stop, resume, pause), and modifying VM vCPU and Memory.
+
+Advanced virtual guest management includes fast refresh, VM lifecycle actions (delete, reset, power off), modifying VM disk, network, graphical display, and graphical display configuration.
```

Comment:
@cbosdo is it right to say Advanced virtual guest management includes everything in the Basic list as well?

Or are those separate?

Reply:
Advanced means Basic + VM console, delete, edit + Virtual nets + Virtual storage